app/src/main/java/ai/elimu/appstore/onboarding/LicenseNumberActivity.java

```diff
@@ -141,12 +142,12 @@ public void onResponse(Call<ResponseBody> call, Response<ResponseBody> response)
                                     startActivity(intent);
                                     finish();
                                 } else {
-                                    // TODO: display error message
+                                    Toast.makeText(getApplicationContext(), "License validation failed", Toast.LENGTH_SHORT).show();
```

Comment:
Please move hard-coded string to string resource. It's easier for future modification.

Reply:
Fixed